### DISCUSSION TITLE
Geostress Balance and Loading Settings

### DISCUSSION THREAD

**Discussion by kuailebozai** [Anura3D/Anura3D_OpenSource#32] [General]
Hello everyone：
I am using anura 3d-2019.1b version. I modeled the pile-bearing embankment, but I can't complete it in the first step for ground stress balance. And I don't know how to continue to add embankment fill after the first step is completed (as shown in the picture in the second step to apply blue and yellow part of fill).
I hope someone can help me to solve the problem, thank you very much! 
![Effective stress](https://user-images.githubusercontent.com/52518762/184539556-8798b313-3525-49b7-917e-3bba38af6d98.png)
![Loading steps](https://user-images.githubusercontent.com/52518762/184539558-c0e473f2-bbf3-43a5-8720-8ef88f62ea26.png)

**Comment by murphy-john** [comment]
What method are you using to initialize the stresses? Quasi-static or the K0 method.

**Reply by kuailebozai** [reply]
I am using the quasi-static method and I have added only gravity loads.Is it possible that the stress cannot be initialized because too much soil material is used.Do you have any method you recommend using? Thank you a lot for your help.

**Comment by ayerrocolom** [comment]
Hello,
Anura3D does not have the capability to add material when the calculation is started. 
To initialize the stresses, you should be able to use the quasi-static method. What error do you get?
